sdk/extensions/Azure.Extensions.AspNetCore.DataProtection.Blobs/src/Azure.Extensions.AspNetCore.DataProtection.Blobs.csproj

```diff
@@ -13,9 +13,8 @@
 
   <ItemGroup>
     <PackageReference Include="Azure.Core" />
-    <PackageReference Include="Microsoft.AspNetCore.DataProtection" />
+    <PackageReference Include="Microsoft.AspNetCore.DataProtection" VersionOverride="8.0.11"/>
     <PackageReference Include="Azure.Storage.Blobs" />
-    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" VersionOverride="8.0.0"/>
```

Comment:
I'd put this back in.   Though we should see the conflict resolved because the more direct reference wins, I think we'll see it come back when we shift to a central package reference again.   

It'll be a good reminder that we need to bump the central version, and we should leave the issue open to track that.